blocks/contact/contact.js

```diff
@@ -1,83 +1,103 @@
+  const profileHeadline = document.createElement('div');
+  profileHeadline.classList.add('headline', 'hl-xs', 'profile__headline');
+
+  const span = document.createElement('span');
+  span.classList.add('headline__eyebrow', 'text--eyebrow');
+  span.textContent = placeholders.contactheadline;
+
+  const heading3 = document.createElement('h3');
+  const innerSpan = document.createElement('span');
+  innerSpan.classList.add('headline__main');
+  innerSpan.textContent = item.Name;
+  heading3.appendChild(innerSpan);
+
+  profileHeadline.appendChild(span);
+  profileHeadline.appendChild(heading3);
+
+  if (item.Org) {
+    const heading4 = document.createElement('h4');
+    heading4.classList.add('headline__sub', 'hl--sub');
+    heading4.textContent = item.Org;
+    profileHeadline.appendChild(heading4);
+  }
+
+  profileItem.appendChild(profileHeadline);
+}
+
+function addListItem(ariaLabel, href, target, title, spanClassList, spanLabel, ul) {
+  const li = document.createElement('li');
+  li.classList.add('profile__contact-item');
+
+  const anchorTag = document.createElement('a');
+  anchorTag.classList.add('plain-link', 'profile__contact-item-link');
+  anchorTag['aria-label'] = ariaLabel;
+  anchorTag.href = href;
+  anchorTag.title = title;
+
+  if (target) {
+    anchorTag.target = target;
+  }
+
+  const spanPhone = document.createElement('span');
+  spanClassList.forEach((x) => {
+    spanPhone.classList.add(x);
+  });
+
+  const spanPhoneLabel = document.createElement('span');
+  spanPhoneLabel.classList.add('profile__contact-item-label');
+  spanPhoneLabel.textContent = spanLabel;
+
+  anchorTag.appendChild(spanPhone);
+  anchorTag.appendChild(spanPhoneLabel);
+
+  li.appendChild(anchorTag);
+  ul.appendChild(li);
+}
+
+function setProfileContact(profileItem, placeholders, item) {
+  const profileContact = document.createElement('div');
+  profileContact.classList.add('profile__contact');
+
+  const ul = document.createElement('ul');
+  ul.classList.add('profile__contact-list');
+
+  addListItem(placeholders.contacttelephon, `tel:${item.Phone}`, '', placeholders.contacttelephon, ['icon', 'icon-phone'], item.Phone, ul);
+  addListItem(placeholders.contactemail, `mailto:${item.Email}`, '', placeholders.contactemail, ['icon', 'icon-mail', 'icon--symbol'], item.Email, ul);
+  addListItem('vCard', item.vCard, '_blank', 'vCard', ['icon', 'icon-file-download'], placeholders.contactvcard, ul);
+  profileContact.appendChild(ul);
+  profileItem.appendChild(profileContact);
+}
+
+function template(block, items, placeholders) {
+  const div = document.createElement('div');
+
+  const profileDetails = document.createElement('div');
+  profileDetails.classList.add('profile-collection__settings', 'grid__container', `primary-count-${items.length}`);
+
+  const innerProfileDetailsDiv = document.createElement('div');
+  innerProfileDetailsDiv.classList.add('grid__structure', 'profile-collection__structure', 'profile-collection__structure--primary');
```

Comment:
Is it possible to get rid of these 3 nested divs as well and directly put the markup inside the `block` div?

Reply:
Will check and is possible will remove this

Reply:
Thanks. Let me also explain my reasoning behind this.
If we remove these nested divs and also the exception for `contact` block [from here](https://github.com/hlxsites/zeiss/blob/ea20c954e14e4de68e28ef1f20dba78ee5b6e677/styles/simplified-styles.css#L151-L159) then the block itself could provide the grid layout. The advantage would be that this article-list block would inherit the global styles applicable for all the blocks thus providing a consistent styling for all the blocks in terms of margins, paddings etc.